packages/faustwp-core/src/getProps.ts

```diff
@@ -11,31 +11,41 @@ import isObject from 'lodash/isObject.js';
 import { DEFAULT_ISR_REVALIDATE } from './getWordPressProps.js';
 import { addApolloState, getApolloClient } from './client.js';
 
-export interface GetNextServerSidePropsConfig<Props = Record<string, unknown>> {
+type Props = Record<string, unknown>;
+
+export interface GetNextServerSidePropsConfig<TProps = Props> {
   Page: {
     query?: DocumentNode;
     variables?: (
       context: GetStaticPropsContext | GetServerSidePropsContext,
+      extra?: TProps,
```

Comment:
Would there be any advantage to using the [rest parameters](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/rest_parameters) with these changes rather than a literal `extra`?

```
      ...extraOptions: TProps[],
```

Reply:
JavaScript function arguments are named but it doesn't matter when you are using them:
This is still valid:
```
Page.variables = ({}, custom) {
}
```